src/accordion/styled-components.js

```diff
@@ -64,6 +64,7 @@ export const ToggleIcon = styled<SharedStylePropsArgT>('svg', props => {
   const {$theme, $disabled, $color} = props;
   return {
     ...getSvgStyles(props),
+    flex: 'none',
```

Comment:
Can you please update to longhand `flex-shrink`? Doing so will help avoid styletron warnings regarding mixed shorthand and longhand properties.

```suggestion
    flexShrink: 0,
```